### PR TITLE
Fill KymaConfig after upgrade

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

- add code to get KymaConfig from reconciler.Cluster

**Related issue(s)**

#1096

### CODE REVIEW

File: components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go [hunk 1]
```diff
@@ -203,6 +203,10 @@ func (h *orchestrationHandler) getOperation(w http.ResponseWriter, r *http.Reque
 		h.log.Errorf("while getting runtime state for upgrade operation %s: %v", operationID, err)
 	}
 
+	if upgradeState.ClusterSetup != nil {
+		upgradeState.KymaConfigFromClusterSetup()
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go:207]: do we want to run this for upgrades not connected to any "orchestration" as well maybe?
https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/broker/instance_update.go

**Review by wozniakjan — DISMISSED** [review]
/lgtm
/hold

feel free to unhold at your convenience

File: components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go (hunk 1 above)
Comment (szwedm) [reply components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go:207]: it depends on whether other endpoints are showing KymaConfig in a response, like this one: https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/common/orchestration/dto.go#L147

Comment (wozniakjan) [reply components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go:207]: good point, I think they don't

File: components/kyma-environment-broker/internal/model.go [hunk 2]
```diff
@@ -407,6 +407,38 @@ type RuntimeState struct {
...
+	}
+
+	profile := gqlschema.KymaProfile(r.ClusterSetup.KymaConfig.Profile)
+	kymaConfig := gqlschema.KymaConfigInput{
+		Version:    r.ClusterSetup.KymaConfig.Version,
+		Profile:    &profile,
+		Components: components,
+	}
+
+	r.KymaConfig = kymaConfig
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/model.go:439]: I'd like to see fields to be immutable, let's discuss

Comment (szwedm) [reply components/kyma-environment-broker/internal/model.go:439]: I will change the method to return KymaConfig without modifying RuntimeState as we discussed

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**